lib/commands/web-view/set-web-view.ts

```diff
@@ -0,0 +1,58 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import commandParams = require("../../common/command-params");
+import semver = require("semver");
+
+export class SetWebViewCommand implements ICommand {
+	constructor(private $webViewService: IWebViewService,
+		private $injector: IInjector,
+		private $errors: IErrors,
+		private $logger: ILogger,
+		private $project: Project.IProject) { }
+	
+	public execute(args: string[]): IFuture<void> {
+		return (() => {		
+			this.$webViewService.enableWebView(args[0], args[1]).wait();
+			this.$logger.out("WebView has been successfully changed.");
+		}).future<void>()();
+	}
+	
+	public canExecute(args: string[]): IFuture<boolean> {
+		return (() => {
+			this.$project.ensureCordovaProject(); 			
+			
+			if(!args[0] || !args[1]) {
+				this.$errors.fail(`Platform and WebView name are mandory params.`);
+			}
+						
+			let supportedWebViews = this.$webViewService.supportedWebViews;
+			
+			// Validate platform
+			let platform = args[0].toLowerCase();
+			let platforms = _.keys(supportedWebViews);
+			if(!_.contains(platforms, platform)) {
+				this.$errors.failWithoutHelp(`Invalid platform. The valid platforms that support webView are: ${platforms.join(", ")}`);
```

Comment:
`Invalid platform. You can set the web view for the following platforms: ${platforms.join(", ")}`